src/view/configuration/streamingMediaSection.jsx

```diff
@@ -12,38 +12,58 @@ governing permissions and limitations under the License.
 
 import React from "react";
 import PropTypes from "prop-types";
-import { number, object, string } from "yup";
+import { number, object, string, lazy } from "yup";
 import { useField } from "formik";
 import SectionHeader from "../components/sectionHeader";
 import FormElementContainer from "../components/formElementContainer";
 import FormikTextField from "../components/formikReactSpectrum3/formikTextField";
 import FormikNumberField from "../components/formikReactSpectrum3/formikNumberField";
 import isNonEmptyString from "../utils/isNonEmptyString";
 
-const getDefaultSettings = () => {
-  return {
-    channel: "",
-    playerName: "",
-    appVersion: "",
-    adPingInterval: 10,
-    mainPingInterval: 10,
-  };
-};
+const getDefaultSettings = () => ({
+  channel: "",
+  playerName: "",
+  appVersion: "",
+  adPingInterval: 10,
+  mainPingInterval: 10,
+});
+
 export const bridge = {
   getInstanceDefaults: () => ({
     streamingMedia: getDefaultSettings(),
   }),
+
   getInitialInstanceValues: ({ instanceSettings }) => {
     if (!instanceSettings.streamingMedia) {
-      instanceSettings.streamingMedia = getDefaultSettings();
+      return bridge.getInstanceDefaults();
     }
 
-    return instanceSettings;
+    const streamingMedia = Object.keys(getDefaultSettings()).reduce(
+      (acc, k) => {
+        if (instanceSettings.streamingMedia[k] !== undefined) {
+          acc[k] = instanceSettings.streamingMedia[k];
+        } else {
+          acc[k] = "";
+        }
+
+        return acc;
+      },
+      {},
+    );
+
+    return { streamingMedia };
   },
+
   getInstanceSettings: ({ instanceValues }) => {
     const instanceSettings = {};
     const { streamingMedia } = instanceValues;
 
+    ["appVersion", "adPingInterval", "mainPingInterval"].forEach((key) => {
+      if (!streamingMedia[key]) {
```

Comment:
Here I clear the empty values of optional fields. Before, they were saved to the DB.